app/src/behaviors/behavior_sticky_key.c

```diff
@@ -21,71 +21,70 @@
 
 LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
-#if DT_NODE_EXISTS(DT_DRV_INST(0))
+#if DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT)
 
 #define ZMK_BHV_STICKY_KEY_MAX_HELD 10
 
-// increase this if you need more keys in the board
-#define ZMK_BHV_STICKY_KEY_POSITION_NOT_USED ULONG_MAX
+#define ZMK_BHV_STICKY_KEY_POSITION_FREE ULONG_MAX
 
 struct behavior_sticky_key_config {
     u32_t release_after_ms;
     struct zmk_behavior_binding behavior;
 };
 
-struct active_sticky_key {
+typedef struct {
     u32_t position;
     u32_t param1;
     u32_t param2;
     const struct behavior_sticky_key_config *config;
     // timer data.
     bool timer_started;
+    bool timer_cancelled;
     s64_t release_at;
     struct k_delayed_work release_timer;
-    bool timer_is_cancelled;
     // usage page and keycode for the key that is being modified by this sticky key
     u8_t modified_key_usage_page;
     u32_t modified_key_keycode;
-};
+} active_sticky_key;
 
-struct active_sticky_key active_sticky_keys[ZMK_BHV_STICKY_KEY_MAX_HELD] = {};
+active_sticky_key active_sticky_keys[ZMK_BHV_STICKY_KEY_MAX_HELD] = {};
 
-static struct active_sticky_key *store_sticky_key(u32_t position, u32_t param1, u32_t param2,
-                                                  const struct behavior_sticky_key_config *config) {
+static active_sticky_key *store_sticky_key(u32_t position, u32_t param1, u32_t param2,
+                                           const struct behavior_sticky_key_config *config) {
     for (int i = 0; i < ZMK_BHV_STICKY_KEY_MAX_HELD; i++) {
-        if (active_sticky_keys[i].position != ZMK_BHV_STICKY_KEY_POSITION_NOT_USED ||
-            active_sticky_keys[i].timer_is_cancelled) {
+        active_sticky_key *sticky_key = &active_sticky_keys[i];
```

Comment:
```suggestion
        active_sticky_key * const sticky_key = &active_sticky_keys[i];
```

Minor (and if you do accept this, clang-format it first), but putting a `const` after the `*` ensures that `sticky_key` can't be reassigned to point to something else later.